### PR TITLE
Changed table headers and added payroll column

### PR DESCRIPTION
**Pull request by Amy-Xu**
Checked items in #62 are included in this PR. Everything is in line with [this PR](https://github.com/OpenSourcePolicyCenter/Tax-Calculator/pull/392) to taxcalc.

### CODE REVIEW

File: webapp/apps/taxbrain/helpers.py [hunk 1]
```diff
@@ -162,8 +165,8 @@ def string_to_float_array(s):
     'df_dec': 'Difference between Base and User plans by expanded income decile',
     'mX_bin': 'Base plan tax vars, weighted avg per expanded income bin',
     'mY_bin': 'User plan tax vars, weighted avg per expanded income bin',
-    'df_bin': 'Difference between Base and User plans by expanded income bin',
-    'fiscal_tots': 'Total Revenue Change by Calendar Year',
+    'df_bin': 'Individual Income Tax: Difference between Base and User plans by AGI decile',
```
Comment (talumbau) [inline webapp/apps/taxbrain/helpers.py]: Hmm.. something has gone wrong here. Any name that ends with `_bin` refers to data that has been grouped by an income bin. Here, you are saying that `df_bin` is referring to data that is grouped by AGI_decile. If I understand correctly, the line should be:
```
'df_bin': 'Individual Income Tax: Difference between Base and User plans by expanded income bin',
```

@Amy-Xu do you see what I mean?

Comment (Amy-Xu) [inline webapp/apps/taxbrain/helpers.py]: Yea, haven't been able to double check with @MattHJensen but I think you're right. Will change this in a sec.

Comment (MattHJensen) [inline webapp/apps/taxbrain/helpers.py]: Yes, should be expanded income bin for now. That said, I hope we can generalize this later on so the user can decide whether to use _expanded_income or c00100 as the tab variable for tables. https://github.com/OpenSourcePolicyCenter/webapp-public/issues/35#issuecomment-148811619

**Comment by talumbau** [comment]
this looks good. Merging.

**Pull request merged**